modules/gitops-synchronizing-cluster-configs-application.adoc

```diff
@@ -0,0 +1,19 @@
+// Module included in the following assemblies:
+//
+// * configuring-an-openshift-cluster-with-argo-cd.adoc
+
+[id="synchronizing-cluster-configs-application_{context}"]
+= Synchronizing `cluster configs` application
+
+. In the Argo CD dashboard, notice that the `cluster-configs` Argo CD application has the statuses `Missing` and `OutOfSync`. Because the application was configured with a manual sync policy, Argo CD does not sync it automatically.
```

Comment:
`cluster-configs` `Missing` and `OutOfSync` must be bold and not in back ticks. Any GUI element that you see is in bold. If the user is to type it out then use back ticks.